examples/vanilla-ts-no-ip/src/custom-sw.ts

```diff
@@ -3,6 +3,9 @@ import { registerRoute } from 'workbox-routing'
 import { CacheFirst, NetworkFirst, StaleWhileRevalidate } from 'workbox-strategies'
 import { CacheableResponsePlugin } from 'workbox-cacheable-response'
 import { ExpirationPlugin } from 'workbox-expiration'
+import orderBy from 'lodash-es/orderBy.js'
+
+console.log(orderBy)
```

Comment:
This may not be needed?

Reply:
It is a test for custom build esbuild, check vite issue

Reply:
okay